tests/Nelmio/Alice/ReferenceRangeNameTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *
+ * (c) Nelmio <hello@nelm.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nelmio\Alice;
+
+class ReferenceRangeNameTest extends \PHPUnit_Framework_TestCase
+{
+    const USER = 'Nelmio\Alice\support\models\User';
+    const GROUP = 'Nelmio\Alice\support\models\Group';
+    const CONTACT = 'Nelmio\Alice\support\models\Contact';
+
+
+    public function testLoadFixturesByReferenceNotFound()
+    {
+        $om = $this->getDoctrineManagerMock(null);
+
+        $files = [
+            __DIR__ . '/support/fixtures/reference_range_2.yml',
+        ];
+        try {
+            Fixtures::load($files, $om, [ 'providers' => [ $this ] ]);
+        } catch (\UnexpectedValueException $e) {
```

Comment:
instead of having this `try/catch` block, you can add `@expectedException \UnexpectedValueException` to the method PHPDoc